### PR TITLE
Functionalities

### PR DESCRIPTION
**Pull request by GuptaEesh**
- added edit,delete,generate task
- added done, undone feature
- added tag filter.
- added dark and light mode
- added data persistence using local storage

[Preview Post Link](https://www.linkedin.com/posts/eesh-gupta-42673a144_hello-connections-recently-i-was-working-activity-6921366257128603648-JhHB/?utm_source=linkedin_share&utm_medium=member_desktop_web)

### CODE REVIEW

**Review by prachi-sahani — COMMENTED** [review]
Code is well structured and good to merge 

Suggestions:
- remove {" "}
- consider formatting your code

**Review by FarhanMobashir — CHANGES_REQUESTED** [review]
Code is neat. Can be merged
Just make sure your code is properly formatted

File: src/Screens/focus-screen/focus-screen.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      </span>
+      <div className="focus-screen padding-1 flex flex-column align-center justify-center">
+        <span>
+          Task Name :- <span className="bold">{task.name}</span>
+        </span>
+
+        <span>
+          Mode :-
+          <span className="bold" style={{ color: bgMoodColor }}>
+            {" "}
```
Comment (FarhanMobashir) [inline src/Screens/focus-screen/focus-screen.js]: Like here

File: src/Screens/focus-screen/focus-screen.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            {bgTitle}
+          </span>{" "}
```
Comment (FarhanMobashir) [inline src/Screens/focus-screen/focus-screen.js]: Here also

File: src/Screens/focus-screen/focus-screen.js (hunk 1 above)
Comment (GuptaEesh) [reply src/Screens/focus-screen/focus-screen.js]: Done. It is sometimes automatically added sorry for that.

**Review by FarhanMobashir — APPROVED** [review]

**Pull request merged**